lib/ansible/plugins/__init__.py

```diff
@@ -97,6 +97,9 @@ def get_options(self, hostvars=None):
             options[option] = self.get_option(option, hostvars=hostvars)
         return options
 
+    def get_option_default(self, option: str) -> t.Any:
```

Comment:
Plz avoid `Any`. MyPy recommends using `object` in case you can't be more specific.
```suggestion
    def get_option_default(self, option: str) -> object:
```

Reply:
https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object

Reply:
we seem to use `any` in most places, is this a new policy?

Reply:
@bcoca I think it's the lack of education in this area. I only learned about this recommendation last summer, I think. And I use typing *a lot*. Many people are still new to it. So I'm trying to bring new things from the outside Python community when I get a chance...